tests/unittests/alltest.py

```diff
@@ -0,0 +1,87 @@
+import unittest
+import sys
+from cypari2 import Pari, PariError
+
+pari = Pari()
+
+testmodules = [
+    'bern',
+    'set',
+    'bit',
+    'characteristic',
+    'charpoly',
+    'chinese',
+    'contfrac',
+    'disc',
+    'ellmodulareqn',
+    'factorint',
+    'galpol',
+    'idealappr',
+    'isprime',
+    'lambert',
+    'lex',
+    'lindep',
+    'linear',
+    'list',
+    'log',
+    'mathnf',
+    'minim',
+    'minmax',
+    'modfun',
+    'modular',
+    'nfrootsof1',
+    'norm',
+    'number',
+    'pol',
+    'prec',
+    'prime',
+    'primes',
+    'qfb',
+    'qfbclassno',
+    'qfsolve',
+    'subcyclo',
+    'sumdedekind',
+    'sumformal',
+    'zeta'
+    ]
+
+require_galpol = ["galpol"]
+require_seadata = ["ellmodulareqn"]
+
+galpol_installed = True
+seadata_installed = True
+
+# test extensions presence.
```

Comment:
I would propose to encapsulate these in functions `has_seadata`, `has_galpol`, `has_elldata`, etc that would actually be part of the cypari2 library. Perhaps as methods of `Pari`.